data/json/vehicle_parts.json

```diff
@@ -2068,6 +2068,26 @@
             "all" : 10
         }
     },
+    {
+        "type" : "vehicle_part",
+        "id" : "plating_acidchitin",
+        "name" : "biosilicified chitin plating",
+        "symbol" : ")",
+        "color" : "yellow",
+        "broken_symbol" : ")",
+        "broken_color" : "light_gray",
+        "durability" : 750,
+        "item" : "acidchitin_plate",
+        "difficulty" : 2,
+        "location" : "armor",
+        "flags" : ["ARMOR"],
+        "breaks_into" : [
+            {"item": "acidchitin_piece", "count": [6, 19]}
+        ],
+        "damage_reduction" : {
+            "all" : 12
+        }
```

Comment:
It should not be better for non-acid-related purposes than regular chitin plate.

Reply:
It's denser and more durable than regular chitin, so they wouldn't be the exact same in damage resistance.